beets/mediafile.py

```diff
@@ -2005,6 +2007,44 @@ def update(self, dict):
         out_type=float,
     )
 
+    # EBU R128 fields.
+    r128_track_gain = MediaField(
+        MP3DescStorageStyle(
+            u'R128_TRACK_GAIN'
+        ),
+        MP3DescStorageStyle(
+            u'r128_track_gain'
+        ),
```

Comment:
It belatedly occurs to me to recommend that we don't try use capitalizations here. It was useful for RG, where there are apparently both styles in use, but here we're essentially "inventing a standard"—I don't see any prior art yet indicating one or the other. We can always add both later if someone complains.

The only problem is I'm not sure which to choose… maybe lower case?